src/ideal/ideal.jl

```diff
@@ -1468,6 +1469,37 @@ function hilbert_series(I::sideal{spoly{T}}, w::Vector{<:Integer}, Qt::PolyRing)
    return Qt(new_ptr)
 end
 
+@doc raw"""
+    hilbert_series_data(I::sideal{spoly{T}}) where T <: Nemo.FieldElem
```

Comment:
Why is a new function needed? Can't we just modify the existing `hilbert_series` functions?

Reply:
Without a second name Oscar-tests would fail: I do not know how to change Oscar and Singular..jl in one step.